backstop/tests/wizard.html

```diff
@@ -33,175 +33,195 @@
-            <span class="iui-wizards-step-title">Yet another step</span>
-            <span class="iui-wizards-step-circle">
-              <span>6</span>
-            </span>
-          </span>
-
-          <span class="iui-wizards-step-track iui-wizards-step-track-main"></span>
-
-          <span class="iui-wizards-step">
-            <span class="iui-wizards-step-title">Last step</span>
-            <span class="iui-wizards-step-circle">
-              <span>7</span>
-            </span>
-          </span>
-        </div>
 
-        <p class="iui-wizards-steps">
-          <span class="iui-wizards-steps-count">Step 3 of 7:</span>
+      <div class="iui-wizard iui-long">
+        <ol aria-label="Progress">
+          <li class="iui-wizard-step iui-clickable">
+            <div class="iui-wizard-track-content">
+              <span
+                class="iui-wizard-circle"
+                aria-selected="false"
+                tabindex="0"
+              >1</span>
+            </div>
+          </li>
+
+          <li class="iui-wizard-step iui-clickable">
+            <div class="iui-wizard-track-content">
+              <span
+                class="iui-wizard-circle"
+                aria-selected="false"
+                tabindex="0"
+              >2</span>
+            </div>
+          </li>
+
+          <li
+            class="iui-wizard-step iui-current"
+            aria-current="true"
+          >
+            <div class="iui-wizard-track-content">
+              <span
+                class="iui-wizard-circle"
+                aria-selected="true"
+              >3</span>
+            </div>
+          </li>
+
+          <li class="iui-wizard-step">
+            <div class="iui-wizard-track-content">
+              <span
+                class="iui-wizard-circle"
+                aria-selected="false"
+              >4</span>
+            </div>
+          </li>
+
+          <li class="iui-wizard-step">
+            <div class="iui-wizard-track-content">
+              <span
+                class="iui-wizard-circle"
+                aria-selected="false"
+              >5</span>
+            </div>
+          </li>
+
+          <li class="iui-wizard-step">
+            <div class="iui-wizard-track-content">
+              <span
+                class="iui-wizard-circle"
+                aria-selected="false"
+              >6</span>
+            </div>
+          </li>
+
+          <li class="iui-wizard-step">
+            <div class="iui-wizard-track-content">
+              <span
+                class="iui-wizard-circle"
+                aria-selected="false"
+              >7</span>
+            </div>
+          </li>
+        </ol>
+
+        <div class="iui-wizard-steps-label">
+          <span class="iui-steps-count">Step 3 of 7:</span>
           Name of current step
-        </p>
+        </div>
       </div>
+
     </section>
 
     <hr />
-    <h2 id="workflow">Workflow diagram</h2>
 
+    <h2>Workflow diagram</h2>
```

Comment:
Missing `iui-current` on one of the steps.
Also, should upcoming oval step look like a completed step?
![image](https://user-images.githubusercontent.com/36186912/129671241-c613b681-18f4-4fcb-8412-50dbb7dcb5cf.png)

Reply:
This does not look right. There should be no "iui-current" in workflow diagram

Reply:
The workflow diagram is kind of unique (and probably shouldn't be within wizard) as it doesn't have the concept of a current step.  In the previous implementation I was using the `.iui-current` class to apply the styling of all the boxes that weren't first or last.  Now it's using `:first-child` and `:last-child` to do the styling instead.